### ISSUE TITLE
--watch seems to have stopped working

### ISSUE THREAD

**Issue by WeeJeWel** [jorgebastida/awslogs#325]
As the title says. It worked a few days ago. Logs get shown up until now but don't update automatically anymore.

**Comment by koshyviv** [comment]
True, had to manually stop the command and rerun to see new logs.

**Comment by igomez10** [comment]
Same, for now I was able to get through it with the `watch` command in unix

```
watch awslogs get /some/group --start="1m ago"
```

It will not work if you add the `--watch` flag

**Comment by koshyviv** [comment]
This is a good alternative, thanks!

**Comment by WeeJeWel** [comment]
Thanks for the workaround @igomez10. I hope someone can fix awslogs itself though! This workaround has no colors and uses polling.

**Comment by kacirekj** [comment]
Pls fix it
